src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -401,9 +402,19 @@ void damped_harmonic_h(
   for (size_t a = 0; a < SpatialDim + 1; ++a) {
     gauge_h->get(a) = (1. - roll_on_h_init) * gauge_h_init.get(a) +
                       get(h_prefac1) * spacetime_unit_normal_one_form.get(a);
+    // In case we want SpEC-style roll-on of gauge
+    if (UNLIKELY(use_spec_style_rollon)) {
```

Comment:
It is probably better to remove the `UNLIKELY` macro, since the flag will have the same value during the entire run.

Reply:
removed